common/nodejs-utils/src/application.rs

```diff
@@ -30,6 +30,22 @@ pub fn check_for_multiple_lockfiles(app_dir: &Path) -> Result<()> {
     }
 }
 
+/// Checks that a lockfile for a package manager we support exists in an
+/// expected location.
+#[must_use]
+pub fn supported_lockfile_exists(app_dir: &Path) -> bool {
+    for manager in [
+        PackageManager::Npm,
+        PackageManager::Pnpm,
+        PackageManager::Yarn,
+    ] {
+        if app_dir.join(manager.lockfile()).exists() {
+            return true;
+        }
+    }
+    false
```

Comment:
Minor - you could simply this with `any`.  E.g.; 

```suggestion
    [
        PackageManager::Npm,
        PackageManager::Pnpm,
        PackageManager::Yarn,
    ]
    .iter()
    .any(|manager| app_dir.join(manager.lockfile()).exists())
```

Reply:
Thanks - I ended up deleting this function in the latest commit anyways. I've merged it with the multi lockfile check you already had.